test/tests/functional/pbs_reservations.py

```diff
@@ -142,7 +156,6 @@ def degraded_resv_reconfirm(self, start, end, rrule=None, run=False):
 
         self.server.expect(RESV, a, id=rid, interval=1)
 
-    @skipOnCpuSet
```

Comment:
Why are you removing this? This would cause the tests that call this function to fail on cpuset.